### PR TITLE
ref(js): Remove usage of emotion as prop

### PR DESCRIPTION
**Pull request by evanpurkhiser**
This just doesn't play well with emotion 11

### CODE REVIEW

File: static/app/components/actions/actionLink.tsx [hunk 1]
```diff
@@ -35,18 +51,33 @@ export default function ActionLink({
...
+        aria-label={title}
+        className={classNames(className, {disabled})}
+        onClick={disabled ? undefined : onAction}
+        disabled={disabled}
+        {...props}
+      >
+        {children}
+      </StyledAction>
+    );
+  }
```
Comment (priscilawebdev) [inline static/app/components/actions/actionLink.tsx:80]:
```suggestion
  const actionCommonProps = {
    ['aria-label']: title,
    className: classNames(className, {disabled}),
    onClick: disabled ? undefined : onAction,
    disabled,
    children,
    ...props,
  };

  const action =
    type === 'button' ? (
      <StyledActionButton {...actionCommonProps} />
    ) : (
      <StyledAction {...actionCommonProps} />
    );
```

Comment (EvanPurkhiser) [reply static/app/components/actions/actionLink.tsx:80]: Yeah this might work, let me try. I think I might have actually tried doing this

Comment (EvanPurkhiser) [reply static/app/components/actions/actionLink.tsx]: That works!

**Comment by billyvg** [comment]
Is `as` removed in 11, or is there something specific that doesn't play well?

**Comment by EvanPurkhiser** [comment]
They don't remove the prop, but they add half way support for typescript, which was easier to just remove our usage of it instead of try and figure out how to make it play nice

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**